Hubs/ChatHubcs.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Microsoft.AspNetCore.SignalR;
+using OnlineConsulting.Models.ValueObjects.Chat;
+using OnlineConsulting.Models.ViewModels.Chat;
+using OnlineConsulting.Services.Repositories.Interfaces;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace OnlineConsulting.Hubs
+{
+    public class ChatHub : Hub
+    {
+        private readonly IChatRepository _chatRepository;
+
+        public ChatHub(IChatRepository chatRepository)
+        {
+            _chatRepository = chatRepository;
+        }
+
+        public async Task CreateConversationAsync(CreateConversationViewModel createConversationViewModel)
+        {
+            var createConversation = new CreateConversation
+            {
+                Host = createConversationViewModel.Host,
+                Path = createConversationViewModel.Path
+            };
+
+            var conversation = await _chatRepository.CreateConversationAsync(createConversation);
+
+            var createMessage = new CreateMessage
+            {
+                Content = createConversationViewModel.FirstMessage,
+                Conversation = conversation,
+                IsFromClient = true
+            };
+
+            await _chatRepository.CreateMessageAsync(createMessage);
+            await JoinTheGroupAsync(conversation.Id.ToString());
+        }
+
+        public async Task JoinTheGroupAsync(string conversationId)
+        {
+            await Groups.AddToGroupAsync(Context.ConnectionId, conversationId);
+
+            var conversationIdGuid = Guid.Parse(conversationId);
```

Comment:
It will throw exception if string passed to method will not be GUID. You might want to use TryParse and return some kind of graceful message, on the other hand it can stay like this, cause user can't write conversation id by hand, so returning 500 if some one have tampered with data is fine in this case. But it's good to know that such thing will happen.